Vungle/VungleRouter.m

```diff
@@ -715,60 +790,65 @@ - (void)vungleWillShowAdForPlacementID:(nullable NSString *)placementID
 }
 
 - (void)vungleDidShowAdForPlacementID:(NSString *)placementID
+                              eventId:(NSString*)eventID
 {
-    id<VungleRouterDelegate> targetDelegate = [self.delegatesDict objectForKey:placementID];
+    id<VungleRouterDelegate> targetDelegate = [self getFullScreenDelegateWithPlacement:placementID eventID:eventID];
     if ([targetDelegate respondsToSelector:@selector(vungleAdDidAppear)]) {
         [targetDelegate vungleAdDidAppear];
     }
 }
 
 - (void)vungleAdViewedForPlacement:(NSString *)placementID
+                           eventId:(NSString*)eventID
 {
     if (!placementID.length) {
         return;
     }
-
-    id<VungleRouterDelegate> targetDelegate = [self.delegatesDict objectForKey:placementID];
+    
```

Comment:
Might there be an extra indent? Thanks a lot :)

Reply:
Updated.